docs/user-guides/workspace-access/jetbrains.md

```diff
@@ -248,14 +248,14 @@ windows-x64, windows-aarch64, linux-x64, linux-aarch64, osx-x64, osx-aarch64
 
 To install both backends and clients, you will need to run two commands.
 
-**Backends**
+#### Backends
 
 ```shell
 mkdir ~/backends
 ./jetbrains-clients-downloader-linux-x86_64-1867/bin/jetbrains-clients-downloader --products-filter <product-code> --build-filter <build-number> --platforms-filter linux-x64,windows-x64,osx-x64 --download-backends ~/backends
 ```
 
-**Clients**
+#### Clients
```

Comment:
this should be tabs, but I'm leaving it for now since I've got a PR coming to edit the JetBrains doc